scipy/stats/_distribution_infrastructure.py

```diff
@@ -3674,6 +3749,58 @@ def _overrides(method_name):
     return CustomDistribution
 
 
+def _make_distribution_custom(dist):
+    parameters = []
+    support = getattr(dist, 'support')
+
+    for name, info in dist.parameters.items():
+        domain = _RealDomain(endpoints=info['endpoints'],
```

Comment:
Let's give these defaults and make them optional.

Reply:
Can you elaborate? Do you mean the endpoints?

Reply:
I think I was thinking about both `endpoints` and `inclusive`. How about `-inf, inf`, non-inclusive?

Reply:
Sounds good.